bundles/org.openhab.binding.bluetooth.radoneye/src/main/resources/OH-INF/thing/radoneye.xml

```diff
@@ -24,6 +24,11 @@
 				<label>Address</label>
 				<description>Bluetooth address in XX:XX:XX:XX:XX:XX format</description>
 			</parameter>
+			<parameter name="fwVersion" type="integer" min="1" max="2">
```

Comment:
Please also update the default translation file to add an entries for this new parameter.

Reply:
There was no translations file, so i added them. I hope this was correct, could you please have a look at the new `bluetooth.properties` and the `bluetooth_de.properties` files?

Reply:
Please avoid providing translations for different languages, it should be rather done with Crowdin. As the files do not yet exist, if you really want to provide German translations, this is acceptable but a manual action will be required by a Crowdin maintainer to upload them in Crowdin.

You only provided a partial list of entries. Please run "mvn i18n:generate-default-translations" so that your file is generated automatically. This binding is "sub-binding" of bluetooth so maybe the tool will generate a propoerties file with a bad name. Considering the other bluetooth "sub-bindings", I believe you make the good choice with file named bluetooth.properties.

Reply:
> As the files do not yet exist, if you really want to provide German translations, this is acceptable but a manual action will be required by a Crowdin maintainer to upload them in Crowdin.

This would then skip the proof-reading of texts which is also a part of the Crowdin process.

Reply:
The 'bluetooth.properties' file is updated and the 'bluetooth_de.properties' file is removed, thanks to hint with the `mvn i18n:generate-default-translations`.

I have removed the 'This could have impact to battery lifetime' text, the device has no battery.